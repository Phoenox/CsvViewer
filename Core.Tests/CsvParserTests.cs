namespace Core.Tests;

public class CsvParserTests
{
	[Fact]
	public void ParsesCsvContent()
	{
		var csvFile = CsvParser.ToCsvFile(Dummies.CsvFileContent);
		csvFile.Should().MatchSnapshot();
	}
}
