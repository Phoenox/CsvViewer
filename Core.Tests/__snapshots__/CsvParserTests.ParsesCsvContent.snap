{
  "Headers": [
    "foo",
    "bar",
    "bla"
  ],
  "Lines": [
    {
      "Cells": [
        "1",
        "2",
        "3"
      ]
    },
    {
      "Cells": [
        "4",
        "5",
        "6"
      ]
    }
  ]
}
